src/services/EnvironmentVariables.ts

```diff
@@ -82,8 +80,7 @@ export class EnvironmentVariables {
 					!this.SESSION_TABLE || this.SESSION_TABLE.trim().length === 0 ||
 					!this.YOTI_KEY_SSM_PATH || this.YOTI_KEY_SSM_PATH.trim().length === 0 ||
 					!this.TXMA_QUEUE_URL || this.TXMA_QUEUE_URL.trim().length === 0 ||
-					!this.GOVUKNOTIFY_API_KEY_SSM_PATH || this.GOVUKNOTIFY_API_KEY_SSM_PATH.trim().length === 0 ||
-					!this.GOVUKNOTIFY_API || this.GOVUKNOTIFY_API.trim().length === 0) {
```

Comment:
Are we checking if REMINDER_EMAIL_GOVUKNOTIFY_API env variable exists in this file? Shouldnt we replace GOVUKNOTIFY_API with REMINDER_EMAIL_GOVUKNOTIFY_API here?